doc/Setup.md

```diff
@@ -0,0 +1,146 @@
+
+Copy the content of Address to the REST API host and port pasted into the wallet
+
+Copy the contents of Admin Macaroon and paste it into the Macaroon (HEX format) in the wallet
+
+Click Continue to create an account
+
+### Testnet/testing-accounts for development use localhost testnet
+
+For most people who are new to the btc lightning network, starting a test version of the lightning network environment locally is very helpful for developing wallets, so that they can transfer money with confidence.
+
+[Start the lightning network test environment locally and link to the Alby](https://github.com/getAlby/lightning-browser-extension/wiki/Start-the-lightning-network-test-environment-locally-and-link-to-the-Alby)
+
+## Project Structure
+
+```bash
+./lightning-browser-extension
+├── src                     # Source Code
+│   ├── app                     # React UI App
+│   ├── extension               # Browser Extension
+│   ├── common                  # Helpers and utilities used by both the React App and the Browser Extension
+├── static                  # Static Resources
+│   ├── assets                  # Images, logos, etc
+│   └── views                   # Static HTML files
+├── doc                     # Documentation (guidelines, architecture docs, etc)
+├── dist                    # Build
+│   └── development             # Developer Builds (not to be shared)
+│   └── production              # Production Builds
+├── tests                   # E2E tests and related helpers
+└
+```
+
+### Debugging
+
+Most logs are written to the background script. Make sure to "inspect" the background script to see the console. Help for [Chrome](https://developer.chrome.com/docs/extensions/mv3/tut_debugging/), [Firefox](https://extensionworkshop.com/documentation/develop/debugging/)
+
+### :white_check_mark: Tests
+
+#### E2E tests via [playwright](https://playwright.dev) ([using testing-library](https://testing-library.com/docs/pptr-testing-library/intro/))
+
+```bash
+yarn run dev:chrome
+yarn test:e2e
+```
+
+:tipping_hand_woman: For now we only do E2E tests for Chrome
+
+#### Unit tests tests via [Jest](https://jestjs.io)
+
+```bash
+yarn test:unit
+```
+
+#### Run all tests
+
+```bash
+yarn test
+```
+
+### ⌨️ Production package files
+
+- `yarn run package` builds the extension for all the browsers to `dist/production` directory respectively.
+
+You can also use a Docker container and run the yarn commands within a container:
+
+```bash
+docker run --rm --volume="$(pwd):/app" --workdir="/app" -t -i node:lts "yarn install && yarn run package"
+```
+
+Note: By default the `manifest.json` is set with version `0.0.0`. The webpack loader will update the version in the build with that of the `package.json` version. In order to release a new version, update version in `package.json` and run script.
+
+## Native Companions
```

Comment:
I think all of this can go to contribution apart from
- Quick setup as mentioned above
- "Native Companions" should go to the main readme to let people know Alby supports Tor

Reply:
Okay, let me modify it to see the effect